src/movement/thruster_sabertooth.cpp

```diff
@@ -117,5 +149,12 @@ int main(int argc, char **argv)
 
   ros::spin();
 
+  uint8_t serial_data[4];
```

Comment:
Did a quick google search and found realized that passing the array this way isn't actually copying the array as it is naturally passed as a pointer. Disregard my previous suggestion to save copying.